frontend/src/metabase/visualizations/echarts/cartesian/model/dataset.ts

```diff
@@ -330,6 +331,24 @@ const getStackedAreasInterpolateTransform = (
   };
 };
 
+function getOtherSeriesTransform(
+  groupedSeriesKeys: DataKey[],
+): ConditionalTransform {
+  return {
+    condition: groupedSeriesKeys.length > 0,
+    fn: datum => {
+      const transformedDatum = { ...datum };
+
+      transformedDatum[OTHER_DATA_KEY] = groupedSeriesKeys.reduce(
+        (sum, key) => sum + checkNumber(datum[key] ?? 0),
```

Comment:
@cdeweyx grouping by SUM works for count, but if series show averages the grouped "Other" value will look misleading

Reply:
Let's ask about this on slack, this is not a very visible discussion forum.